content/docs/user-guide/gems/reference/aws/aws-core/configuring-credentials.md

```diff
@@ -227,6 +227,38 @@ To set up a team, repeat the instructions for individual users above to:
 
 Please read [Working with AWS credentials](#working-with-aws-credentials) to decide the right method for providing AWS IAM credentials for your O3DE project.
 
+## Running your O3DE project on Amazon EC2
+
+If you are running your project on Amazon EC2, you have the additional option of using the underlying [instance profile](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html#ec2-instance-profile) to authenticate calls made to AWS. Using the EC2 instance role lets you avoid needing to separately configure AWS credentials on the machine through less secure methods like setting the AWS environment variables through user data at deploy time or manually remoting into the machine to set up a profile.
+
+To use EC2 instance role credentials with your project:
+1. Create an EC2 [instance profile](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-ec2_instance-profiles.html) through the [Amazon web console](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-ec2_instance-profiles.html#instance-profiles-manage-console) or AWS CLI. An instance profile is essentially a container for an IAM role that your EC2 instance can assume to make calls to AWS.
+1. Provide the associated IAM role any [permissions](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_manage-attach-detach.html) required to access the AWS resources your O3DE project needs.
+1. Attach the instance profile to the EC2 instance(s) running your O3DE project. You can attach it to a new instance [at launch time](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/LaunchingAndUsingInstances.html) or you can [attach it to a running instance](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html#attach-iam-role). 
+
+The AWS Core Gem also requires that the `AllowAWSMetadataCredentials` setting be set to `true` before it will query the [EC2 instance metadata service (IMDS)](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html) for credentials. This is to avoid uselessly calling this endpoint when running your O3DE project locally or on untrusted client machines. 
```

Comment:
```suggestion
The AWS Core Gem also requires that the `AllowAWSMetadataCredentials` setting be set to `true` before it will query the [EC2 instance metadata service (IMDS)](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html) for credentials. Use this setting is to prevent unwanted calls to the Amazon EC2 IMDS endpoint when running your O3DE project locally or on non Amazon EC2 compute. 
```

Reply:
will update, but prefer to say:
> **This default setting** is to prevent unwanted...

..."use" makes me think you need to take some action to prevent these calls, which you don't b/c default

Reply:
Solid point, always good to be clear, feel free to adjust my language, just wanted to avoid "uselessly calling" as it sounded too strong.